projects/2021_22_05_pkt_wallet_website_GUI_for_the_blockchain.md

```diff
@@ -127,7 +127,7 @@ __Add the option to announcement mine on from your browser.__
 > * It will be possible to announcement mine from within your account on the website. 
 
 #### Payout
-After a report is delivered on this milestone and it is approved by the NS, the NS will send 4,000,000 PKT.
+After a report is delivered on this milestone and it is approved by the NS, the NS will send 11,000,000 PKT.

```

Comment:
Note that the proposal can only be considered in the form which was proposed as of the time of the deadline for applications.

Reply:
Oke I understand, we are looking forward to the results of the current round!